src/sentry/integrations/opsgenie/client.py

```diff
@@ -22,3 +32,23 @@ def __init__(self, base_url: str, api_key: str) -> None:
     def get_account(self):
         headers = {"Authorization": "GenieKey " + self.api_key}
         return self.get(path="/account", headers=headers)
+
+
+class OpsgenieClient(ApiClient):
```

Comment:
Now that we're starting on the real client (no longer the Setup Client) you can import the `IntegrationProxyClient` and set that up. You can find the [usage docs here](https://github.com/getsentry/sentry/blob/master/src/sentry/shared_integrations/client/integration_proxy_client.md), but ping me if you need anything specific since it's a hybrid cloud thing :)